core/src/main/scala/com/rallyhealth/vapors/factfilter/data/FactTable.scala

```diff
@@ -0,0 +1,50 @@
+package com.rallyhealth.vapors.factfilter.data
+
+import com.rallyhealth.vapors.core.data.Indexed
+
+final case class FactTable private (private val facts: Map[String, List[Fact]]) {
```

Comment:
I'm not sure if there should be some definition for how to sort the facts that are in the FactTable (this could have a potentially large impact on performance).

Reply:
I opted for making this a SortedSet which sorts facts by fact type name in a subsequent PR. I will also investigate how to sort facts within a single type.